src/config/settings.js

```diff
@@ -19,6 +19,13 @@ class Settings {
     };
 
     this.servicePort = this.getEnviroment('PORT');
+
+    this.jwt ={
+      jwtSecret: this.getEnviroment('SECRET'),
+      expiresIn: this.getEnviroment('EXPIRESIN'),
```

Comment:
Não seria bom separar as palavras com _?
```suggestion
      expiresIn: this.getEnviroment('EXPIRES_IN'),
```

Reply:
Corrigido